Cython/Compiler/ExprNodes.py

```diff
@@ -2879,12 +2879,12 @@ def generate_result_code(self, code):
 class ImportNode(ExprNode):
     #  Used as part of import statement implementation.
     #  Implements result =
-    #    __import__(module_name, globals(), None, name_list, level)
+    #    __import__(module_name, globals(), None, list(name_tuple), level)
     #
     #  module_name   UnicodeNode            dotted name of module. Empty module
     #                       name means importing the parent package according
     #                       to level
-    #  name_list     ListNode or None      list of names to be imported
+    #  name_tuple     TupleNode or None      list of names to be imported
```

Comment:
```suggestion
    #  name_tuple    TupleNode or None      list of names to be imported
```
But … maybe it could be called `imported_names`?